xdc-plugin/xdc.cc

```diff
@@ -385,9 +384,25 @@ struct ReadXdc : public Frontend {
 		extra_args(f, filename, args, argidx);
 		std::string content{std::istreambuf_iterator<char>(*f), std::istreambuf_iterator<char>()};
 		log("%s\n", content.c_str());
+
+		// According to page 6 of UG903 XDC is tcl, hence quuoting of bracketed numbers,
+		// such as bus indexes, is required. However, it's quite common for EDA tools
+		// to allow for correct processing of the bracketed numbers without quoting.
+		// Possible TCL implementations of such a feature include registering a TCL command
```

Comment:
This needs explicit examples.  E.g.

> For example, tcl would typical expand:
>
>> signal[5]
>
> as the string "signal" concatenated with the function call "5" with no arguments.

Then explain in detail how the modification to the unknown function handles this.

Reply:
Done

Reply:
Good, thanks!